### ISSUE TITLE
Selecting Multiple Assignees Does "AND" instead of "OR"

### ISSUE THREAD

**Issue by andreamwilhelm** [waffleio/waffle.io#2654]
With a recent update, waffle.io no longer supports "AND" when selecting multiple users and seeing all issues on both. 

For example, when tracking my QA team, I want to see all issues assigned to Andrea, Adrienne, and Alex, yet now the filter is only looking for issues with Andrea, Adrienne, AND Alex assigned all on each ticket.

- with one person selected: ![spreemo_chewy](https://cloud.githubusercontent.com/assets/17707304/16239784/812e51da-37b4-11e6-873e-c0350d65157b.png)
- with more than one person selected: ![spreemo_chewy](https://cloud.githubusercontent.com/assets/17707304/16239818/9e19d602-37b4-11e6-97c1-e2b3869533dd.png)
- with more than one person when those people share a ticket: ![spreemo_chewy](https://cloud.githubusercontent.com/assets/17707304/16239842/c4b9ac92-37b4-11e6-92ac-35baa38a55d1.png)

- [ ] Support "OR" for selecting multiple assignees in waffle.io

**Comment by cabloo** [comment]
I can't even select multiple users any more - it just switches.

**Comment by kellihsf** [comment]
Hi @andreamwilhelm,

Thanks for opening this issue! When we implemented multiple assignees, we decided to use the same filtering that we do for labels, which is using AND not OR when filtering by multiple components. We don't have any immediate plans for changing this, but I'll keep it open for other users to find. Thanks! :)

**Comment by kellihsf** [comment]
@cabloo,

Can you give me a little more detail on what switches when you're filtering by multiple assignees? Thanks!

**Comment by cabloo** [comment]
@kellihsf

First of all, the logic has changed within the last week. It used to be an
OR filter, which was much more convenient for small teams. If I were to
click on unassigned then my name, if would show both unassigned AND my
cards.

Now, if I click on my name and then another name, and it just switches the
filter  to the other person. Same if I try holding shift or ctrl.

Additionally, my bookmark that contained ?assigned=unassigned,cabloo does
nothing.

Thanks,
Zane

**Comment by macdonaldr93** [comment]
@kellihsf our team is having the same issue. We can only filter with one value in the `assigned` parameter, otherwise, it displays nothing.

`?assigned=unassigned,macdonaldr93` displays nothing
`?assigned=unassigned` shows unassigned
`?assigned=macdonaldr93` shows to my username

**Comment by lukaszgryglicki** [comment]
Is there any chance that it will be supported again.

**Comment by kellihsf** [comment]
Hi everyone, thanks for all of the feedback on this! We've decided that we'd like to support this filtering again. Feel free to subscribe to the issue in GitHub to follow it's activity.  :)

**Comment by davidrans** [comment]
Sounds like you guys have already decided to fix this, but this issue hasn't been updated in a while. My Waffle workflow is totally broken until I can select issues assigned to any of a group of people.

**Comment by darryn** [comment]
Any movement on this @kellihsf ?

**Comment by kellihsf** [comment]
Sorry everyone, no immediate plans on this one yet. We'll move it to our 'Planned' column when it gets scheduled into an iteration!

**Comment by macdonaldr93** [comment]
@kellihsf that's too bad. I can't even really use waffle until this is reverted...

**Comment by jtgrenz** [comment]
Yeah,. I've pretty much given up on waffel at this point and I'm using a series of bookmarks and chrome plugins and the new github projects. `¯\_(ツ)_/¯`

**Comment by lasseebert** [comment]
I miss this feature too. My use case is that I want to see tickets assigned to me or not assigned to anyone.

**Comment by linkmeyer** [comment]
@kellihsf , please add this functionality back ASAP.  Then, add the boolean OR capability when searching with other fields as well.  There are many times I want to be able to specify OR logic in my queries, but can't because everything is always AND'd together.  This is very problematic and limiting.
